test/js/src/test/scala/zio/test/TestPlatform.scala

```diff
@@ -2,10 +2,14 @@ package zio.test
 
 import scala.concurrent.{ ExecutionContext, Future }
 
-object ExitUtils {
-
-  def fail(): Unit = System.exit(-1)
+private[test] object TestPlatform {
 
   def await(f: Future[Boolean])(implicit ec: ExecutionContext): Unit =
     f.foreach(_ => ())
+
+  def fail(): Unit = System.exit(-1)
+
+  val isJS = true
+
+  val isJVM = false
```

Comment:
Would like to see these two helpers (`isJS` and `isJVM`) inside `ZIOTest`, so we could make `aspects` and a combinator for platform-specific tests.

```scala
test("Javascript-only") { ... } @@ js
```